client/account-recovery/reset-code-validation/index.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+import qs from 'qs';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+
+import {
+	updatePasswordResetUserData,
+	setResetMethod,
+	validateRequest,
+	validateRequestError,
+} from 'state/account-recovery/reset/actions';
+
+import {
+	getAccountRecoveryValidationError,
+	getAccountRecoveryValidationKey,
+} from 'state/selectors';
+
+class ResetPasswordEmailValidation extends Component {
+	getQueryString = () => {
+		const queryString = get( window, 'location.search', null );
+
+		if ( ! queryString ) {
+			return '';
+		}
+
+		return queryString.slice( 1 );
+	}
+
+	parseQueryArgs = () => ( qs.parse( this.getQueryString() ) );
```

Comment:
Nope. It was an accident :D 
Updated in https://github.com/Automattic/wp-calypso/pull/13450/commits/ea57a2c7aed39a87a07bb823b81ce812148cf9ef#diff-fbc0e17344f63b1a5fb3da56568e5dc1L38